app/views/open_desk/central_navigation/menu.html.erb

```diff
@@ -1,20 +1,27 @@
 <%= turbo_frame_tag "opendesk-central-navigation" do %>
   <% @entries.each do |group| %>
-    <ul class="op-opendesk-navigation--group">
-      <li class="op-opendesk-navigation--item-header"><%= group.name %></li>
+    <%= render Primer::Alpha::ActionList.new(classes: "op-opendesk-navigation--group") do |list| %>
+      <% if group.name.present? %>
+        <% list.with_heading(title: group.name, align_items: :flex_start, classes:"op-opendesk-navigation--item-header") %>
+      <% end %>
+
       <% group.items.each do |item| %>
-        <li class="op-opendesk-navigation--item">
-          <%= link_to item.link, class: "op-opendesk-navigation--item-link op-menu--item-action" do%>
+        <% list.with_item(
+          href: item.link,
+          label: nil,
+          test_selector: "op-menu--item-action",
+          classes: "op-opendesk-navigation--item op-opendesk-navigation--item-link op-menu--item-action"
+        ) do |menu_item| %>
+          <div class="d-flex align-items-center gap-2" style="height: 24px;">
             <% if item.icon.present? %>
-              <%= tag :img,
-                      src: item.icon,
-                      class: "op-opendesk-navigation--item-icon"
-              %>
+              <%= tag.img src: item.icon,
+                          class: "op-opendesk-navigation--item-icon",
+                          alt: "#{item.name} icon" %>
             <% end %>
-            <span><%= item.name %></span>
-          <% end %>
-        </li>
+            <span style="padding-top: 2px;"> <%= item.name %> </span>
+          </div>
```

Comment:
These inline styles are really ugly. Please do that with a class and set the height on the image and the span directly. Then you don't need a magic top padding which would break as soon as people change the default font size (which some browsers do).